tests/unit/base-route.spec.ts

```diff
@@ -0,0 +1,172 @@
+          PAY_API_VERSION: '/pay api version',
+          REGISTRY_HOME_URL: 'registry home url',
+          SENTRY_DSN: 'sentry dsn',
+          SENTRY_ENABLE: 'sentry enable',
+          SITEMINDER_LOGOUT_URL: 'siteminder logout url',
+          STATUS_API_URL: 'status api url',
+          STATUS_API_VERSION: '/status api version'
+        }
+      }))
+  })
+
+  afterEach(() => {
+    sinon.restore()
+  })
+
+  it('fetches and loads the configuration to the store', async () => {
+    // mock window.location getters
+    delete window.location
+    window.location = {
+      origin: 'http://localhost',
+      pathname: '/business/CP1234567',
+      search: '?accountid=2288'
+    } as any
+
+    // call method
+    const store = await getVuexStore() as any // remove typings for unit tests
+    const applicationUrl = 'http://localhost/business/'
+    setBaseRouteAndBusinessId('CP1234567', '/business/', window.location.origin)
+    await store.dispatch('fetchConfiguration', applicationUrl)
+      .then(() => {
+        nextTick()
+        // verify data
+        expect(store.getters.getAddressCompleteKey).toBe('address complete key')
+        expect(store.getters.getAuthApiUrl).toBe('auth api url/auth api version/')
+        expect(store.getters.getAuthWebUrl).toBe('auth web url')
+        expect(store.getters.getCreateUrl).toBe('business create url')
+        expect(store.getters.getEditUrl).toBe('business edit url')
+        expect(store.getters.getBusinessFilingLdClientId).toBe('business filing ld client id')
+        expect(store.getters.getBusinessUrl).toBe('businesses url')
+        expect(store.getters.getKeycloakConfigPath).toBe('keycloak config path')
+        expect(store.getters.getLegalApiUrl).toBe('legal api url/legal api version 2/')
+        expect(store.getters.getPayApiUrl).toBe('pay api url/pay api version/')
+        expect(store.getters.getRegHomeUrl).toBe('registry home url')
+        expect(store.getters.getSentryDsn).toBe('sentry dsn')
+        expect(store.getters.getSentryEnable).toBe('sentry enable')
+        expect(store.getters.getSiteminderLogoutUrl).toBe('siteminder logout url')
+        expect(store.getters.getStatusApiUrl).toBe('status api url/status api version')
+      })
+  })
+
+  it('fetches and loads the configuration to session variables', async () => {
+    // mock window.location getters
+    delete window.location
+    window.location = {
+      origin: 'http://localhost',
+      pathname: '/business/CP1234567',
+      search: '?accountid=2288'
+    } as any
+
+    // call method
+    const store = await getVuexStore() as any // remove typings for unit tests
+    const applicationUrl = 'http://localhost/business/'
+    setBaseRouteAndBusinessId('CP1234567', '/business/', window.location.origin)
+    await store.dispatch('fetchConfiguration', applicationUrl)
+
+    // verify data
+    // expect(sessionStorage.getItem('AUTH_API_URL')).toBe('auth api url/auth api version/')
```

Comment:
Please delete obsolete code.

Reply:
Done, will be visible after the next push

Reply:
You mean INvisible :)